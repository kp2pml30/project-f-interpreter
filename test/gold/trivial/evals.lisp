(eval 1)
(eval 2.5)
(eval false)
(eval true)
(eval null)
(eval (cond true 1 0))

(setq b (quote (break)))
(while true
    (print (quote loop))
    (eval b))
