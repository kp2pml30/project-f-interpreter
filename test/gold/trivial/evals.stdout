"int:1"
2.5
false
true
"int:1"
"loop"
